packages/dx-react-grid/docs/guides/searching.md

```diff
@@ -0,0 +1,29 @@
+# React Grid - Searching
```

Comment:
Maybe. But I think no because we named feature filtering in here:
https://github.com/DevExpress/devextreme-reactive/blob/master/packages/dx-react-grid/docs/guides/filtering.md#react-grid---filtering.

Reply:
@viterobk

Reply:
I'd name it searching, like in DevExtreme: https://js.devexpress.com/Documentation/Guide/Widgets/DataGrid/Filtering_and_Searching/